### PR TITLE
Fix post info to not overflow

### LINKED ISSUE #2568: Min-height for blog posts needs adjustment to avoid GitHub details overflowing into the next post
**What happened**:
When a user submits a short blog, the blog details on the right side overflow into the next blog.

**What should have happened**:
The blog details should not overflow into the blog below - we should try to set a min-height per blog to avoid the overflow. This will leave some extra space in the blog area, but will not interfere with the design.

**How to reproduce it (as precise as possible)**:
You need to find a short blog, here I have attached an image for example:
image

**Anything else we need to know?**:
There's already a min-height set. I am sure this was set before we implemented the GitHub details and therefore needs to be adjusted. We need to explore what the maximum height of the GitHub details will be(repo, issues, pull requests, users etc.), and consider that when finding the sweet spot for the min-height. Below is a screenshot at min-height of 400px from the 300px we have right now:
image

**Environment**:

- OS:
- Browser:

### PR DESCRIPTION
**Pull request by Kevan-Y**
## Issue This PR Addresses

 Fixes #2568 
## Type of Change

- [X] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description

The post info overflow the post container if the post text wasn't long enough.
I set the root to `display: flow-root` so it will makes sure everything will stay in that post container.

image

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: src/web/src/components/Posts/Post.tsx [hunk 1]
```diff
@@ -29,14 +29,14 @@ type Props = {
 const useStyles = makeStyles((theme: Theme) =>
   createStyles({
     root: {
+      display: 'flow-root',
```
Comment (humphd) [inline src/web/src/components/Posts/Post.tsx:32]: Fascinating!  I've never heard of it before, TIL.  Nice work.

**Review by humphd — APPROVED** [review]

**Review by DukeManh — APPROVED** [review]
So cool, it works so well, til.
image

**Comment by humphd** [comment]
@Kevan-Y land this whenever you are ready.  Nice work.

**Comment by AmasiaNalbandian** [comment]
I'm late to the party but I wanted to add that this is a VERY nice fix. Well done!

**Pull request merged**